R/spark_shell.R

```diff
@@ -7,6 +7,68 @@ shell_connection_validate_config <- function(config) {
   config
 }
 
+databricks_connection <- function(config, extensions) {
```

Comment:
I would create a new file for `databricks_connection` named `R/databricks_connection.R`